src/api/gpu/ucx/nixl_device.cuh

```diff
@@ -106,7 +172,33 @@ nixlGpuPostPartialWriteXferReq(nixlGpuXferReqH *req_hndl,
                                bool is_no_delay = true,
                                nixlGpuXferStatusH *xfer_status = nullptr)
 {
-    return NIXL_ERR_NOT_SUPPORTED;
+    uint64_t flags                    = is_no_delay ? UCP_DEVICE_FLAG_NODELAY : 0;
+    ucp_device_request_t *ucp_request = nullptr;
+
+    if (!req_hndl) {
+        return NIXL_ERR_INVALID_PARAM;
+    }
+
+    if (xfer_status != nullptr) {
+        ucp_request = &xfer_status->device_request;
+        flags      |= UCP_DEVICE_FLAG_COMP;
+    }
+
+    if (signal.inc != 0) {
```

Comment:
1. IMO better check if signal.remote_addr != 0
2. also no need to define special flag, we can say UCP also does not post signal if the signal remote address is 0